packages/x-charts-pro/src/context/ZoomProvider/useSetupZoom.ts

```diff
@@ -89,6 +90,13 @@ export const useSetupZoom = () => {
       }
 
       event.preventDefault();
+      if (interactionTimeoutRef.current) {
+        clearTimeout(interactionTimeoutRef.current);
+      }
+      setIsInteracting(true);
+      interactionTimeoutRef.current = window.setTimeout(() => {
+        setIsInteracting(false);
+      }, 166);
```

Comment:
```suggestion
      // Debounce transition to `isInteractive=false`. 
      // Useful because wheel events don't have an "end" event.
      interactionTimeoutRef.current = window.setTimeout(() => {
        setIsInteracting(false);
      }, 166);
```